_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -73272,9 +73274,11 @@
 /obj/machinery/atmospherics/components/unary/outlet_injector/on{
 	dir = 1;
 	frequency = 1441;
-	id = "n2_in"
+	id = "n2_in";
+	volume_rate = 200
 	},
-/turf/open/floor/plating/airless,
+/obj/structure/lattice,
+/turf/open/space/basic,
```

Comment:
please use `/turf/open/space`

Reply:
I really REALLY wish I could change it in the Github editor, ugh

Reply:
notepad.

Reply:
^
very tiny map changes can be done in a text editor instead of a laggy map editor